packages/ui/src/TextField.tsx

```diff
@@ -279,8 +279,6 @@ export function TextField({
                 width: "100%",
                 color: Unifier.theme.darkGray,
                 fontFamily: Unifier.theme.primaryFont,
-                // Remove border in web.
```

Comment:
Unrelated?

Reply:
This is causing the mobile apps to crash due to this property being deprecated. I should have placed a TODO on here to add the styles to mimic this behavior.

Reply:
Oh interesting, and weird!